build-system/tasks/visual-diff/index.js

```diff
@@ -595,6 +598,14 @@ async function ensureOrBuildAmpRuntimeInTestMode_() {
   }
 }
 
+function installPercy_() {
+  log('info', 'Running', colors.cyan('yarn'), 'to install Percy...');
+  execOrDie('npx yarn --cwd build-system/tasks/visual-diff');
```

Comment:
Suggestion for a future cleanup PR: Silence the `yarn` logs on Travis by calling ` execOrDie('npx yarn --cwd build-system/tasks/visual-diff', {'stdio': 'ignore'});`